### ISSUE TITLE
Incorrect duplicate key registered when using YAML anchors

### ISSUE THREAD

**Issue by ChristopherSchubert** [redhat-developer/vscode-yaml#82]
I receive a "Duplicate Key" error highlight and tooltip when reference multiple YAML anchors at the same level within an element.

For example, defining two different anchors, which do not share the same keys:

```
x-anchor-one: &anchor_one
    sub-property-one: "one"
x-anchor-two: &anchor_two
    sub-property-one: "two"
```

causes the error when referenced as:

```
object:
  property:
    <<: &anchor_one
    <<: &anchor_two
```

Full snippet:
```
x-anchor-one: &anchor_one
    sub-property-one: "one"
x-anchor-two: &anchor_two
    sub-property-one: "two"

object:
  property:
    <<: &anchor_one
    <<: &anchor_two
```

Screenshot:
![yaml_anchor_error](https://user-images.githubusercontent.com/18542093/38884917-234583b2-423f-11e8-8b1b-652f595f4a60.png)

**Comment by JPinkney** [comment]
Fixed in 0.0.12 release.

**Comment by MostafaWael-CA** [comment]
I am still getting the same error in v1.15.0
